app/components/primer/heading_component.rb

```diff
@@ -1,23 +1,46 @@
 # frozen_string_literal: true
 
 module Primer
-  # Use the Heading component to wrap a component that will create a heading element
+  # Use Heading to communicate page organization and hierarchy.
+  #
+  # ## Usage
+  #
+  # - Use as the title of a section or sub section. 
+  # - Use one `<h1>`` per page. 
+  # ### Do not
+  # - Skip heading levels. For instance, do not jump from `<h1>` to `<h3>`.
+  # - Use Heading for sole stylistic purposes. For styling text, consider using [Text] with relevant
+  # [typography arguments].
+  #
+  # [Text]: https://primer.style/view-components/components/text
+  # [typography arguments]: https://primer.style/view-components/system-arguments#typography
```

Comment:
Since we run YARD comments through ERB, we could add helper methods to the Rakefile that return these URLs and call them here, in case we want to change the URLs across the library in the future 😄